hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyRenameRequestWithFSO.java

```diff
@@ -103,8 +103,8 @@ public void testValidateAndUpdateCacheWithToKeyInvalid() throws Exception {
   @Test
   public void testValidateAndUpdateCacheWithEmptyToKey() throws Exception {
     String emptyToKeyName = "";
-    OMRequest omRequest = doPreExecute(createRenameKeyRequest(volumeName,
-        bucketName, fromKeyName, emptyToKeyName));
+    OMRequest omRequest = createRenameKeyRequest(volumeName,
+        bucketName, fromKeyName, emptyToKeyName);
```

Comment:
hmm. if we don't call `doPreExecute()` here, then it won't actually execute the logic in `validateAndUpdateCache` right? doesn't match the test name if that is the case